service/SpaceCenter/src/Services/Camera.cs

```diff
@@ -96,6 +97,45 @@ public void NextCamera()
             CameraManager.Instance.NextCamera();
         }
 
+        /// <summary>
+        /// Switch to the previous available camera
+        /// </summary>
+        [KRPCMethod]
+        public void PreviousCamera()
+        {
+            var cameraManager = CameraManager.Instance;
+            switch(cameraManager.currentCameraMode)
+            {
+                case CameraManager.CameraMode.Flight :
+                    if (InputLockManager.IsUnlocked(ControlTypes.CAMERAMODES))
+                    {
+                        var camera = FlightCamera.fetch;
+                        if (camera.targetMode != FlightCamera.TargetMode.Vessel || camera.vesselTarget != FlightGlobals.ActiveVessel)
+                        {
+                            camera.SetTargetVessel(FlightGlobals.ActiveVessel);
+                        }
+                        // There is 5 camera modes in the enum.
+                        // As KSP1's development has ceased its cheaper to hardcode it than compute it each time. - Sofie 10-07-2024
```

Comment:
Let me know if you disagree with this sentiment and want it changed out

Reply:
Fine by me, as long as mods can't add additional cameras?

Reply:
As its an enum, even if a mod could do so it wouldn’t be able to be added to the existing enum as they’re locked in at compile time. But i’ll verify if enum’s can be harmony patched. In which case i’ll modify this

Reply:
It seems Harmony cannot patch members only methods so the only in which this enum could change would be if another version of KSP was to be released.
The only other concern regarding this entire added method is if some other mod added a harmony prefix to the existing KSP1 `NextCamera` method. In that case kRPC would rightfully call the patched method when executing the `NextCamera` but kRPC’s reversed copy for the `PreviousCamera` would not contain the changes the patch applied. 
there’s nothing that can realistically be done about this. If such an issue ever arrises either kRPC or more likely the mod making changes the the `NextCamera` method will need to be adjusted.

Reply:
OK great. This sounds fine to me then